scripts/rabbitmq-server-ha.ocf

```diff
@@ -1445,7 +1445,7 @@ get_monitor() {
       fi
     fi
 
-    if [ $rc -eq $OCF_ERR_GENERIC -o $rc_check -eq $OCF_ERR_GENERIC ]; then
+    if [ "$rc" -eq "$OCF_ERR_GENERIC" -o "${rc_check:-$OCF_SUCCESS}" -eq "$OCF_ERR_GENERIC" ]; then
```

Comment:
What default value should be used? `$OCF_SUCCESS`? Isn't it conterintuitive, given that default value for `rc` is `$OCF_ERR_GENERIC`?